consumer-test/TestRunnerThread.groovy

```diff
@@ -47,9 +47,12 @@ class TestRunnerThread extends Thread {
                         "--message=Set piper lib version for test"])
 
         executeShell("docker run -v /var/run/docker.sock:/var/run/docker.sock " +
-            "-v ${System.getenv('PWD')}/${testCaseWorkspace}:/workspace -v /tmp -e " +
-            "CASC_JENKINS_CONFIG=/workspace/jenkins.yml -e CX_INFRA_IT_CF_USERNAME -e " +
-            "CX_INFRA_IT_CF_PASSWORD -e BRANCH_NAME=${testCase} ppiper/jenkinsfile-runner")
+            "-v ${System.getenv('PWD')}/${testCaseWorkspace}:/workspace " +
+            "-v /tmp " +
+            "-e CASC_JENKINS_CONFIG=/workspace/jenkins.yml " +
+            "-e ${System.getenv('CX_INFRA_IT_CF_USERNAME')} " +
```

Comment:
I doubt this is what we want to do. This will pass `-e myusername -e mypassword`, right? The idea with `-e CX_INFRA_IT_CF_USERNAME -e CX_INFRA_IT_CF_PASSWORD` was to avoid writing the password to the process list.

Reply:
Could you explain to me the use of these variables. Are they not environment variables. How is it working currently, as these variables are not gathered from the environment?!

Reply:
Those variables are used to configure the credentials for SCP in Jenkins, cf https://github.com/SAP/jenkins-library/blob/master/consumer-test/jenkins.yml#L24

What I wanted to avoid is to write the username and password in this file (for obvious reasons), and the config as code plugin supports supplying values via environment variables. Not sure if that answers the question.

Reply:
I think I get it. So about your previous comment:
> I would probably still document the required environment `CX_INFRA_IT_CF_USERNAME` and `CX_INFRA_IT_CF_PASSWORD`.

I could do a quick check that these environment variables are set, else I output a helpful message?

Reply:
> output a helpful message

sure